run_cpu.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+
+# prepare conda tool, mortgage dataset
+. ./prereq_install.sh
+
+conda env create -f requirements_cpu_daal.yml
+echo "env mortgage_cpu_daal is created"
+
+echo "conda activate mortgage_cpu_daal"
+conda activate mortgage_cpu_daal
+echo "mortgage_cpu_daal is activated"
+
+# build pandas from source
```

Comment:
this should be no longer needed as they've finally released 0.25

Reply:
We no longer want to get the actual performance numbers?